### PR TITLE
sig-pm: Add project-board-maintainers team

### PR DESCRIPTION
**Pull request by justaugustus**
Signed-off-by: Stephen Augustus <saugustus@vmware.com>

/assign @calebamiles @idvoretskyi @jdumars

### CODE REVIEW

**Comment by cblecker** [comment]
What’s the intent here as far as permissions?

**Comment by justaugustus** [comment]
@cblecker -- I've added an issue to discuss project grooming here: https://github.com/kubernetes/community/issues/3267

Basically, groups that wanted their backlogs groomed would add `project-managers` to their board with write access. Ideally, I'd love if that would happen automatically for org-level boards, but I'm not sure if that's possible in GitHub.

**Comment by cblecker** [comment]
two nit picky things (aka naming is hard):
- "project-managers" has a more generic implication, rather than "a team of people who are being given permissions to board". maybe something like "sig-pm-board-maintainers" would be more explicit as to what the team is actually doing
- The description kind of implies "this group must be given permissions to boards" as opposed to "this is a team of people who can sigs can subscribe to, to help them with maintaining project boards". The latter is a bit wordy, but again, maybe there is a way to be more clear.

**Comment by cblecker** [comment]
/lgtm
/approve

Looks great, @justaugustus!

**Comment by justaugustus** [comment]
@cblecker -- thanks for the tips! Updated team name to `project-board-maintainers` and updated description.

**Comment by justaugustus** [comment]
/hold cancel
(Ihor's hold was around the communication aspect of it, but this has now been opened for discussion in kubernetes/community#3267)

**Comment by idvoretskyi** [comment]
> /hold cancel
> (Ihor's hold was around the communication aspect of it, but this has now been opened for discussion in [kubernetes/community#3267](https://github.com/kubernetes/community/issues/3267))

+1 🎉
